assets/chat/js/chat.js

```diff
@@ -635,6 +635,13 @@ class Chat {
       .catch(() => {});
   }
 
+  setPreLoginText() {
+    const chatText = ChatStore.read('chat.preLoginText') || '';
+    this.input.val(chatText);
```

Comment:
You clear the value of the input when there isn't any pre-login text in the store. This can result in a scenario where any text the user enters vanishes during a reconnect, which sends another `ME` event and retriggers `setPreLoginText()`.

Reply:
addressed